djes/signals.py

```diff
@@ -0,0 +1,10 @@
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+
+from .models import Indexable
+
+
+@receiver(post_delete)
+def delete_es_index_on_delete(sender, instance, **kwargs):
+    if isinstance(instance, Indexable):
```

Comment:
@mparent61 
just to explain what is going on here,

you can't explicitly state a sender for abstract base classes, so instead we listen to all delete calls and interpret from there. Could be pretty wasteful, but most of our classes are Indexable, so it shouldn't be an issue.